README.md

```diff
@@ -182,7 +182,9 @@ Example:
 - LinkedIn: [monicah-wanjema](https://www.linkedin.com/in/monicah-wanjema/)
 
 👤 **Author2**
-
+- GitHub: [@AbdifatahYasin1](https://github.com/AbdifatahYasin1)
```

Comment:
Hi @monicah79 and @AbdifatahYasin1 

- [ ] Please the Table of Contents should be above the Getting Started section. Try to correct that

- [ ] Kindly add the command for **Install**, **Usage**, **Run tests**, **Deployment** headers.

- [ ] The link to your **MIT License** file is broken.

Reply:
Hi but on the requirements, we were asked not to add no files that were nt included in the template...and the previous reviewer asked me to remove the deployment since the project wasnt going to be deployed

Reply:
Yeah you can remove the deploy link